app/assets/javascripts/wikis.js

```diff
@@ -8,7 +8,8 @@ function setupWiki(node_id, raw, logged_in) {
       selector: '#content-raw-markdown',
       wysiwyg: true,
       preProcessor: preProcessMarkdown,
-      postProcessor: postProcessContent
+      postProcessor: postProcessContent,
+      extraButtons: {"fa-comment": true}
```

Comment:
Ah, this is looking good, but is it perhaps not ready until we actually have a function that this button will do? I think a good place to start will be to integrate this into an example in https://github.com/publiclab/inline-markdown-editor, showing that when you click the button, something runs (it could even be a simple `alert(some_output_here)` that the user can see working in the test. 

Then, we'd add to this PR a function that would run to actually show a comment form. Until then, I think we don't want to actually merge this PR in, but it's great to see it here since it will definitely be an important step once the comment form actually works. 

So, what's the first step in getting a commenting form to appear? Let's think through where we'd want to make the form (shown here: https://github.com/publiclab/plots2/blob/master/app/views/comments/_form.html.erb) appear. 

Even before that, maybe we want to create a functional test of adding a comment to a wiki page. This could be a good project to break out and make a checklist out of. Comment controller is functionally tested here: https://github.com/publiclab/plots2/blob/master/test/functional/comment_controller_test.rb#L20-L30

Is there anything stopping us from making a comment attached to a wiki page instead? Could we test that out in the comment controller functional test, as a first step?